frameworks/Java/javalin/build.gradle

```diff
@@ -0,0 +1,44 @@
+apply plugin: 'java'
+apply plugin: 'eclipse'
```

Comment:
The eclipse plugin isn't required for running the benchmark, is it?  If so, can you please remove this line and the Eclipse section at the bottom of this file?  We try to keep the code in TFB the minimum required to run the benchmark, so stuff like functional tests and IDE configuration aren't included here even if they are useful in real-world projects.

Reply:
removed.